docs/tools/offline/cli-unittest.md

```diff
@@ -0,0 +1,58 @@
+## Unit testing
```

Comment:
I'm not sure this should be its own page. I think it would make more sense in the CLI test and debug section.